topostats/filters.py

```diff
@@ -315,10 +316,15 @@ def filter_image(self) -> None:
         self.images["initial_tilt_removal"] = self.remove_tilt(self.images["initial_median_flatten"], mask=None)
         self.images["initial_quadratic_removal"] = self.remove_quadratic(self.images["initial_tilt_removal"], mask=None)
 
+        # Remove scars
```

Comment:
The logic for controlling whether to run the filtering should sit here within the `Filter.filter_image()` method as there's no point making the call to `scars.remove_scar()` if its not going to do anything.  

```suggestion
        # Remove scars
        if scar_config["run"]:
            scar_config.pop("run")
```